File: mwrbs-controller/src/main/java/com/itl/mwrbs/models/DefaultValues.java
package com.itl.mwrbs.models;

import java.util.ArrayList;
import java.util.List;

public class DefaultValues {
	
	public List<PaymentType> paymentTypes;
	public List<RoomType> roomTypes;
	public List<UserType> userTypes;
	public List<RoomStatus> roomStatuses;
	public List<ReservationStatus> reservationStatuses;
	
	public DefaultValues() {
		paymentTypes = new ArrayList<PaymentType>();
		roomTypes = new ArrayList<RoomType>();
		userTypes = new ArrayList<UserType>();
		roomStatuses = new ArrayList<RoomStatus>();
		reservationStatuses = new ArrayList<ReservationStatus>();
	}

}

File: mwrbs-frontend/src/app/_interfaces/default-values.ts
export interface DefaultValues {
    paymentTypes: PaymentType[];
    roomTypes: RoomType[];
    userTypes: UserType[];
    roomStatuses: RoomStatus[];
    reservationStatuses: ReservationStatus[];
}

export interface PaymentType {
    paymentTypeId: number;
    paymentTypeName: string;
}

export interface RoomType {
    roomTypeId: number;
    roomTypeName: string;
}

export interface UserType {
    userTypeId: number;
    userTypeName: string;
}

export interface RoomStatus {
    roomStatusId: number;
    roomStatusName: string;
}

export interface ReservationStatus {
    statusId: number;
    statusName: string;
}
